.github/workflows/main.yml

```diff
@@ -18,26 +18,13 @@ jobs:
     - name: Install Dependencies
       run: |
         pip install tox
-    - name: Run lint
+        sudo snap install charm --classic
+    - name: Lint
       run: tox -vve lint
-  unit-test:
-    name: Unit Tests
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python: [3.6, 3.7, 3.8, 3.9]
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2
```

Comment:
Is this not necessary to get the repository checked out for the tox runs?

Reply:
I consolidated the lint & unit (and now the wheelhouse) tasks into a single one to speed up the checks on GH (not blocking on separate VMs) and reduce our resource usage.  It's hidden by the upper fold, but that part is still there in the workflow.

Reply:
Oh wait this is just the diff. There is a checkout it's earlier in the code. You combined lint, unit, wheelhouse so the diff just shows the removal b/c the checkout already existed on the lint step.